roles/sap_ha_set_hana/templates/20-saphana.j2

```diff
@@ -0,0 +1,12 @@
+# Managed by Ansible
```

Comment:
Can this template file be given a clearer filename? Or more explanation on this line of the template's purpose?

I'm a bit confused by the filename prefix of `20-` also

Reply:
The prefix helps with the order in which the files in /etc/sudoers.d/ are included. The file (so far) contains sudo entries for SRHook monitoring. Added some description.

Reply:
Resolved by 3218e06